internal/cli/streams/processor/list.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2025 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package processor
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/cli"
+	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/cli/require"
+	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/config"
+	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/flag"
+	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/store"
+	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/usage"
+	"github.com/spf13/cobra"
+	atlasv2 "go.mongodb.org/atlas-sdk/v20241113004/admin"
+)
+
+type ListOpts struct {
+	cli.ProjectOpts
+	cli.OutputOpts
+	cli.ListOpts
+	cli.StreamsOpts
+	includeStats bool
+	store        store.ProcessorLister
+}
+
+func (opts *ListOpts) Run() error {
+	r, err := opts.store.ListProcessors(opts.ConfigProjectID(), opts.Instance)
+	if err != nil {
+		return err
+	}
+
+	sps := make([]atlasv2.StreamsProcessorWithStats, 0, len(*r.Results))
+	opts.Output = jsonOutput
+	if opts.includeStats {
+		return opts.Print(*r.Results)
+	}
+
+	for _, res := range *r.Results {
```

Comment:
don't use pointers, use getters https://github.com/mongodb/atlas-sdk-go/blob/main/docs/doc_5_best_practices.md#using-getters-instead-of-direct-field-access
```suggestion
	for _, res := range r.GetResults() {
```